es/indexer_test.go

```diff
@@ -96,16 +96,20 @@ type Classification struct {
 }
 
 type Source struct {
-	MDB_UID string `json:"mdb_uid"`
-	Name    string `json:"name"`
+	MDB_UID string     `json:"mdb_uid"`
+	MDB_ID  null.Int64 `json:"mdb_id"`
```

Comment:
Why do you beed to use MDB_ID?

I think we should use them only on specifc way, e.g., just to fetch but never store MDB_IDs as they can change when migrating to different database.....

Prefer using UID.


What do you do with the mdb_id?

Reply:
removed